src/components/subscribe/subscribe.test.ts

```diff
@@ -50,13 +53,15 @@ const expectElementNotToBeVisuallyHidden = async (
 // Helper to initialize the component JS
 const initJS = (utils: PleasantestUtils) =>
   utils.runJS(`
-    import { initSubscribe } from './subscribe'
-    export default () => initSubscribe(
+    import { createSubscribe } from './subscribe';
+    const subscribe = createSubscribe(
       document.querySelector('.js-subscribe')
-    )
+    );
+    subscribe.init();
+    export default () => subscribe;
```

Comment:
I realize this was exported before, but it is not clear to me why. I think this line can be removed?

Reply:
https://github.com/cloudfour/cloudfour.com-patterns/pull/1882/commits/4aca16a04d345f8024851c764dd2d22b9fa63105